### PR TITLE
Jr jb add vendor controller

### PR DESCRIPTION
**Pull request by jraaaaanger**

### CODE REVIEW

File: spec/features/vendors/01_user_can_create_vendor_spec.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    scenario 'I visit the root path and there is a new vendor button' do
+      expect(page).to have_link 'Add New Vendor'
+    end
+
+    scenario 'I add a new vendor correctly' do
+      click_link 'Add New Vendor'
+      expect(page).to have_content('New Vendor Form')
+
+      fill_in('Name', with: 'Union Square Donuts' )
```
Comment (nwalberts) [inline spec/features/vendors/01_user_can_create_vendor_spec.rb]: This needs a quick edit :P

File: spec/features/vendors/01_user_can_create_vendor_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+feature "user sees a vendor form", %{
+  As a user
+  I wanna see a form for vendors
+  So that I can add a vendor
+} do
+  #ACCEPTANCE CRITERIA:
+  # [] Vendor#create goes to vendors/new
+  # []
```
Comment (nwalberts) [inline spec/features/vendors/01_user_can_create_vendor_spec.rb]: I'd snip this real quick as well

File: app/controllers/vendors_controller.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  def new
+    @vendor = Vendor.new
+  end
+
+  def create
+    @vendor = Vendor.new(vendor_params)
+    if @vendor.save
+      redirect_to @vendor, notice: "You have successfully created a new vendor!"
+    else
+      redirect_to new_vendor_path, notice: "Please try again."
```
Comment (CeeNoriega) [inline app/controllers/vendors_controller.rb]: do you want it to re-direct or render? You might want to render :new

File: spec/controllers/vendors_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+require 'rails_helper'
+RSpec.describe VendorsController, type: :controller do
+end
```
Comment (CeeNoriega) [inline spec/controllers/vendors_controller_spec.rb]: why is this test file empty?

File: spec/features/vendors/01_user_can_create_vendor_spec.rb [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  As a user
+  I want to have access to a vendor form
+  So I can add new vendors and see them
+} do
+  # ACCEPTANCE CRITERIA:
+  # [X] Unauthenticated user does not see new vendor link
+  # [X] Authenticated user does see a new vendor link_to
+  # [X] New vendor link takes user to vendor form
+  # [] New vendor form has necessary fields
+  # [] Create vendor button creates vendor and takes user to Vendor#show
```
Comment (CeeNoriega) [inline spec/features/vendors/01_user_can_create_vendor_spec.rb]: delete these before putting in a PR!

File: spec/helpers/vendors_helper_spec.rb [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+# Specs in this file have access to a helper object that includes
+# the VendorsHelper. For example:
+#
+# describe VendorsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```
Comment (CeeNoriega) [inline spec/helpers/vendors_helper_spec.rb]: delete commented out code

File: spec/helpers/vendors_helper_spec.rb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe VendorsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (CeeNoriega) [inline spec/helpers/vendors_helper_spec.rb]: delete file if not in use!

**Pull request merged**